go-controller/pkg/util/gateway-cleanup.go

```diff
@@ -105,3 +96,31 @@ func GatewayCleanup(nodeName string) error {
 	}
 	return nil
 }
+
+func staticRouteCleanup(clusterRouter string, nextHops []string) error {
+	for _, nextHop := range nextHops {
+		// Get a list of all the routes in cluster router with the next hop IP.
+		var uuids string
+		uuids, stderr, err := RunOVNNbctl("--data=bare", "--no-heading",
+			"--columns=_uuid", "find", "logical_router_static_route",
+			"nexthop="+nextHop)
+		if err != nil {
+			return fmt.Errorf("Failed to fetch all routes with"+
+				"IP %s as nexthop, stderr: %q, "+
```

Comment:
Should we log this error and continue? So, that you will remove the static routes associated with another `nexthop` in the array?

Reply:
done